kcidb/monitor/spool/__init__.py

```diff
@@ -116,6 +127,8 @@ def post(self, notification, timestamp=None):
                 message=notification.render().as_string(
                     policy=email.policy.SMTPUTF8
                 ),
+                notification_id=notification.id,
```

Comment:
Why are you adding the notification ID as a field here?

Reply:
It was mentioned on the issue description

> We need to store the notification ID , as well as the message (or maybe better, the message templates), along with a timestamp somewhere (perhaps Firestore again

Reply:
Yeah, I was just throwing a rough idea at the wall, trying to think outside the box, not necessarily using Firestore. Since we're using Firestore so far, we don't need to add a property for notification ID, we're already storing it in the document ID.